ginga/util/wcsmod/wcs_astropy_ape14.py

```diff
@@ -191,7 +191,7 @@ def wcspt_to_datapt(self, wcspt, coords='data', naxispath=None):
         try:
             args = [wcspt[:, i] for i in range(wcspt.shape[1])]
             # NOTE: Ignores system transformation.
-            datapt = np.asarray(self.wcs.world_to_pixel_values(*args))[:, :2].T
+            datapt = np.asarray(self.wcs.world_to_pixel_values(*args))[:2, :].T
         except Exception as e:
```

Comment:
This looks like the critical fix.  The rest is just logging changes, I guess.

Reply:
LGTM, AFAIAC.

Reply:
Yes. This line is the bug fix.